src/NodeAnalyzer/ConstFetchAnalyzer.php

```diff
@@ -11,31 +11,32 @@
  */
 final class ConstFetchAnalyzer
 {
-    public function isBool(Node $node): bool
+    public function isBool(?Node $node): bool
     {
         return $this->isTrue($node) || $this->isFalse($node);
     }
 
-    public function isFalse(Node $node): bool
+    public function isFalse(?Node $node): bool
     {
         return $this->isConstantWithLowercasedName($node, 'false');
     }
 
-    public function isTrue(Node $node): bool
+    public function isTrue(?Node $node): bool
     {
         return $this->isConstantWithLowercasedName($node, 'true');
     }
 
-    public function isNull(Node $node): bool
+    public function isNull(?Node $node): bool
     {
         return $this->isConstantWithLowercasedName($node, 'null');
     }
 
-    private function isConstantWithLowercasedName(Node $node, string $name): bool
+    private function isConstantWithLowercasedName(?Node $node, string $name): bool
```

Comment:
This should not be nullable

Reply:
All of them, or just this private one? See the commit message for an example

Reply:
All of them, or just this private one? See the commit message for an example

Reply:
All, there should never be nullable node
That would make api weak and every method would have to check that

Reply:
I check that but don't understand